modules/openapi/test/resources/foo.json

```diff
@@ -1,699 +1,711 @@
 {
```

Comment:
This is unfortunate that this change is counted like that in git. VSCode is doing a better job visualizing the differences:
![Screenshot from 2025-06-13 08-46-35](https://github.com/user-attachments/assets/8ddeccfa-4a16-48d9-a539-a445f030097f)